src/utils_stackdriver_json.h

```diff
@@ -46,6 +46,19 @@ and collectd_time_series_response_test.json.
 */
 int parse_time_series_summary(char *buffer, time_series_summary_t *response);
 
+/*
+Append an error to a time_series_summary_t instance. `code` will be used as the
+key in the `errors` list.
+*/
+void time_series_summary_append_error(time_series_summary_t *s, int code, int point_count);
+
+/*
+Pairwise add all the elements of `b` into `a`. If both inputs contain an element
+with the same error code, the output will have one element with the sum of the
```

Comment:
This confused me at first. What do you think of adding this clarification:
> the output will have one element **for that error code** with the sum of the point counts.

Reply:
Yeah, that reads better.